components/hab/src/cli.rs

```diff
@@ -1229,9 +1236,22 @@ pub fn sub_sup_bash() -> App<'static, 'static> {
     )
 }
 
+fn config_file_to_defaults(config_file: &str)
+                           -> Result<PartialSubSupRun, Box<dyn std::error::Error>> {
+    let contents = fs::read_to_string(config_file)?;
+    Ok(toml::from_str(&contents)?)
```

Comment:
Nothing to change here (yet?), but I'm sure somebody is going to take issue with TOML as a format (users have done this already in other parts of the code). Since this is all `serde` under the hood, we could probably pretty easily accept config in TOML, JSON, YAML, or whatever.

We can just wait until somebody asks, though, but the fact that it'd be easy to change is nice :smile: